gateway/k8s/deployment.yml

```
@@ -33,4 +33,14 @@ spec:
\ No newline at end of file
+                  key: JWT_SECRET_KEY
+            - name: REDIS_HOST
+              valueFrom:
+                secretKeyRef:
+                  key: REDIS_HOST
+                  name: onevoice-secret
+            - name: REDIS_PORT
+              valueFrom:
+                secretKeyRef:
+                  key: REDIS_PORT
+                  name: onevoice-secret
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)